Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivateForTesting.h

```diff
@@ -52,6 +52,7 @@ struct WKAppPrivacyReportTestingData {
 
 @property (nonatomic, readonly) NSString *_caLayerTreeAsText;
 
+- (NSDictionary *)_propertiesOfLayerWithID:(unsigned long long)layerID;
```

Comment:
maybe this can be typed as `NSDictionary<NSString *, id> *` for a tad more type safety?